geometry_msgs/Pose before
---
geometry_msgs/Pose after
